### PR TITLE
Menu: Add fixed, color and left props

### PR DESCRIPTION
**Pull request by loris**
- Allow `fixed` to be passed a position (`<Menu fixed="left"`)
- Add `color` prop (`<Menu color="blue">`)
- Add `left` prop  (`<Menu left>`)

PS: don’t know why this is changing so many lines, I have the same settings as the `.editorconfig` file:

### CODE REVIEW

File: src/components/views/menu/menu.jsx [hunk 1]
```diff
@@ -78,6 +86,10 @@ export default class Menu extends React.Component {
          */
         pointing: React.PropTypes.bool,
         /**
+         * A menu can be formatted to float left
+         */
+        left: React.PropTypes.bool,
+        /**
          * A menu can be formatted to float right
```
Comment (asvetliakov) [inline src/components/views/menu/menu.jsx]: There is already floated prop, but it turns class name into left floated and right floated. Not sure if left floated/right floated className makes sense, may be fix it then?
i.e ``` classes[this.props.floated] = !!this.props.floated ?

**Comment by asvetliakov** [comment]
Also can you add new props into index.d.ts interface MenuProps ?
Not sure why it's ruining spaces in your editor

File: src/components/views/menu/menu.jsx (hunk 1 above)
Comment (loris) [inline src/components/views/menu/menu.jsx]: Yeah, `left floated` does not exist, just `left` or `right` classes (https://github.com/Semantic-Org/UI-Menu/blob/master/menu.css#L480), fixing it

**Comment by loris** [comment]
Done! Didn’t know about the `index.d.ts` file, is something than can be used in Sublime Text?
By the way, I think the spaces are being ruined by the following settings in ST:
```
	"trim_automatic_white_space": true,
	"trim_trailing_white_space_on_save": true,
```
Will make sure to avoid this from now on

**Comment by asvetliakov** [comment]
Thanks!
You can use https://github.com/Microsoft/TypeScript-Sublime-Plugin for ts files.
I'd recommend using it, especially if you're depending on semantic-react - it will provide for you props validation, in case of changing/moving/renaming

**Pull request merged**